package.json

```diff
@@ -197,8 +197,33 @@
         "command": "puppet-bolt.OpenUserInventoryFile",
         "title": "Open Bolt User Inventory File",
         "category": "Puppet"
+      },
+      {
+        "command": "puppet.refreshFacts",
+        "title": "Refresh",
+        "icon": {
+          "light": "assets/light/sync.svg",
```

Comment:
TODO: Need actual icons instead of placeholder text

Reply:
There should be a nice refresh icon in the vscode list - https://github.com/microsoft/vscode-icons

Reply:
Yep, this was more a reminder to go there while I got this working :)